### PR TITLE
 [MIG] stock_account_operating_unit: Migration to 14.0

### PR DESCRIPTION
**Pull request by RujiaL**

### CODE REVIEW

**Review by alan196 — APPROVED** [review]
:+1: Functional review

File: stock_account_operating_unit/model/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                            {
+                                "journal_id": journal_id,
+                                "line_ids": move_lines,
+                                "company_id": move.company_id.id,
+                                "ref": move.picking_id and move.picking_id.name,
+                                "stock_move_id": move.id,
+                            }
+                        )
+                    )
+                    am.action_post()
```
Comment (alan196) [inline stock_account_operating_unit/model/stock_move.py:134]:
```suggestion
                    am.sudo().action_post()
```

File: stock_account_operating_unit/model/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                    move_lines = move._prepare_account_move_line(
+                        move.product_qty,
+                        move.product_id.standard_price,
+                        acc_valuation,
+                        acc_valuation,
+                        _("%s - OU Move") % move.product_id.display_name,
+                    )
+                    am = (
+                        self.env["account.move"]
+                        .with_context(
```
Comment (alan196) [inline stock_account_operating_unit/model/stock_move.py:120]:
```suggestion
                        .sudo()
                        .with_context(
```



If you don't use this you must give billing access to stock users and some companies don't want this.

Odoo do it in this lines in stock_account module.

https://github.com/odoo/odoo/blob/14.0/addons/stock_account/models/stock_move.py#L455-L464

**Review by alan196 — CHANGES_REQUESTED** [review]

**Comment by gdgellatly** [comment]
Closing this, @RujiaL is no longer working on Odoo. Feel free to resurrect in new PR.

**Pull request closed without merging**